services/api/src/resources/project/resolvers.ts

```diff
@@ -629,8 +629,54 @@ export const updateProject: ResolverFn = async (
 
   const oldProject = await Helpers(sqlClientPool).getProjectById(id);
 
-  // TODO If the privateKey changes, automatically remove the old one from the
+  // If the privateKey is changed, automatically remove the old one from the
```

Comment:
We're not removing the old private key in this PR. Is it still a TODO or are we deciding not to do that? Comment should reflect either option.